SPECS-EXTENDED/perl-Image-Info/perl-Image-Info.spec

```
@@ -0,0 +1,269 @@
+Summary:        Image meta information extraction module for Perl
+Name:           perl-Image-Info
+Version:        1.42
+Release:        4%{?dist}
+License:        GPL+ OR Artistic
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://metacpan.org/release/Image-Info
+Source0:        https://cpan.metacpan.org/authors/id/S/SR/SREZIC/Image-Info-%{version}.tar.gz#/perl-Image-Info-%{version}.tar.gz
+BuildRequires:  coreutils
+BuildRequires:  findutils
+BuildRequires:  make
+BuildRequires:  perl-generators
+BuildRequires:  perl-interpreter
+BuildRequires:  perl(Carp)
+BuildRequires:  perl(Compress::Zlib)
+BuildRequires:  perl(Config)
+BuildRequires:  perl(Exporter)
+BuildRequires:  perl(ExtUtils::MakeMaker)
+BuildRequires:  perl(File::Basename)
+BuildRequires:  perl(File::Spec)
+BuildRequires:  perl(FindBin)
+BuildRequires:  perl(Getopt::Long)
+BuildRequires:  perl(IO::Scalar)
+BuildRequires:  perl(IO::String)
+BuildRequires:  perl(Image::Xbm)
+BuildRequires:  perl(Image::Xpm)
+BuildRequires:  perl(Test::More)
+BuildRequires:  perl(Test::Pod)
+BuildRequires:  perl(Test::Pod::Coverage) >= 1.00
+BuildRequires:  perl(XML::LibXML::Reader)
+BuildRequires:  perl(XML::Simple)
+BuildRequires:  perl(constant)
+BuildRequires:  perl(lib)
+BuildRequires:  perl(overload)
+BuildRequires:  perl(strict)
+BuildRequires:  perl(vars)
+Requires:       rgb
+Requires:       perl(:MODULE_COMPAT_%(eval "`perl -V:version`"; echo $version))
+Requires:       perl(Compress::Zlib)
+Requires:       perl(IO::Scalar)
+BuildArch:      noarch
+
+%description
+This Perl extension allows you to extract meta information from
+various types of image files.
+
+%prep
+%setup -q -n Image-Info-%{version}
+chmod -c 644 exifdump imgdump
+
+%build
+perl Makefile.PL INSTALLDIRS=vendor
+make %{?_smp_mflags}
```

Ideally, I'd update all such entries like so:
```suggestion
%make_build
```

This way we can modify common make arguments through our macros.

---

did this for all :)